R/format.R

```diff
@@ -0,0 +1,50 @@
+data_import_format <- function(object, brief, name) {
+  if (brief) {
+    return("Data import object")
+  }
+  nms <- setdiff(ls(object), c("format", "clone", "initialize"))
```

Comment:
If you add `cloneable = FALSE` to the class definition you won't have to worry about the clone method

Reply:
Removed `"clone"` here.